recipes-example/template-dbus-service/template-dbus-service.bb

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2019 Luxoft Sweden AB
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+
+DESCRIPTION = "Template for creating a D-Bus Service"
+HOMEPAGE = "https://github.com/Pelagicore/template-dbus-service"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=815ca599c9df247a0c7f619bab123dad"
+PV = "1.0+git${SRCREV}"
```

Comment:
this should go away

Reply:
I don't agree. This is yocto standard for recipes, isn't it?
https://www.yoctoproject.org/docs/1.8/ref-manual/ref-manual.html#source-fetching-dev-environment

Reply:
And https://www.yoctoproject.org/docs/1.8/ref-manual/ref-manual.html#examining-build-history-information

Reply:
Add version to the recipe name instead.

Reply:
rename to *_git.bb and keep this as is?

Or hum, should it be `PV = "1.0+git${SRCPV}"` ? (Searched for `git${` and/or `_git.bb` in https://www.yoctoproject.org/docs/latest/mega-manual/mega-manual.html .)

Reply:
I will rename the file to _git.bb (as was recommended in the yocto manual) also I will change to use SRCPV instead of SRCREV for PV.